### PR TITLE
Plans: Tighten up the plans header design

### PR DESCRIPTION
**Pull request by apeatling**
Before:

screen shot 2016-06-28 at 9 58 02 am

After:

screen shot 2016-06-28 at 9 57 42 am

Test live: https://calypso.live/?branch=update/plans-design-tweaks

### CODE REVIEW

**Comment by rickybanister** [comment]
Looks much better

File: client/my-sites/plan-features/style.scss [hunk 1]
```diff
@@ -31,17 +31,17 @@ $plan-features-header-banner-height: 20px;
 .plan-features__header {
 	position: relative;
 	display: flex;
-	align-items: center;
-	padding: 12px 24px 12px 12px;
+	align-items: top;
```
Comment (lamosty) [inline client/my-sites/plan-features/style.scss]: `top` is not a valid property value for `align-items`. Please check out https://css-tricks.com/almanac/properties/a/align-items/ for more info.

**Comment by lamosty** [comment]
:+1: I like this version too. Let's wait for a design review before merging this to prod.

**Comment by apeatling** [comment]
@lamosty I think @rickybanister's review is a thumbs up, so I'm going to go ahead and merge this. I've fixed the invalid property.

**Pull request merged**